src/std/fs.ab

```diff
@@ -108,3 +108,34 @@ pub fun glob_multiple(paths: [Text]): [Text]? {
 pub fun glob(path: Text): [Text]? {
     return glob_multiple([path])?
 }
+
+/// Extract the file detecting from the filename the extension
+/// Supports: bz2, gz, xz, bz2, deb, rar, rpm, tar(gz/xz/bz), zip(war/jar), 7z
+/// Note: Not all the commands supports the output folder path
+pub fun extract(path: Text, target: Text): Bool{
+    if file_exist(path) {
+        if {
+            contains_multiple(path, [".tar.bz2", ".tbz", ".tbz2"]): trust $tar xvjf "{path}" -C "{target}"$
```

Comment:
You can't use `contains_multiple()` here, because of false positives.  Consider this script:
```
hwalters@Ghostwheel ~/git/amber-lang (extract-stdlib) 
$ cat contains.ab 
#!/usr/bin/env amber
import { contains_multiple } from "std/text"

fun test_multiple(text, terms) {
    let result = contains_multiple(text, terms)
    echo "{text} {result}"
}

test_multiple("foo.abc", [".gz", ".zip"])
test_multiple("foo.gz", [".gz", ".zip"])
test_multiple("foo.zip", [".gz", ".zip"])
test_multiple("foo.abc.extra", [".gz", ".zip"])
test_multiple("foo.gz.extra", [".gz", ".zip"])
test_multiple("foo.zip.extra", [".gz", ".zip"])
```
You should not match for `foo.gz.extra` or `foo.zip.extra`:
```
hwalters@Ghostwheel ~/git/amber-lang (extract-stdlib) 
$ ./contains.ab 
foo.abc 0
foo.gz 1
foo.zip 1
foo.abc.extra 0
foo.gz.extra 1
foo.zip.extra 1
```

Reply:
I'm not sure what the best solution is here. I think we probably need a new function `contains_regex()`, which uses `sed` like `replace_regex()` does.